content/applications/finance/accounting/bank/reconciliation/reconciliation_models.rst

```diff
@@ -31,96 +28,96 @@ There are three types of Reconciliation Models:
 ---------------------------------------------
 
-When you are reconciling an entry with an *Open Balance*, you can use the buttons available under
-the *Manual Operations* tab to pre-fill all the values automatically, before validating the
-reconciliation. Each button is a different Reconciliation Model.
+When you are reconciling an entry with an *open balance*, you can use the buttons available under
+the :guilabel:`Manual Operations` tab to pre-fill all the values automatically, before validating
+the reconciliation. Each button corresponds to a reconciliation model.
 
 .. image:: reconciliation_models/reconciliation_models_button.png
    :align: center
-   :alt: Example of a Reconciliation Model with a write-off button in Odoo Accounting
+   :alt: Example of a reconciliation model with a write-off button in Odoo Accounting.
 
 .. _reconciliation_models_suggestion:
 
 Suggest counterpart values
 --------------------------
 
-This type of Reconciliation Model suggests immediately counterpart values that only need to be
-validated. This automation is based on a set of rules defined in the reconciliation model.
+This type of reconciliation model immediately suggests counterpart values when selecting an entry,
+which then only needs to be validated. This automation is based on a set of rules defined in the
+reconciliation model.
 
 .. image:: reconciliation_models/reconciliation_models_suggestion.png
    :align: center
-   :alt: Example of a Reconciliation Model that suggests counterpart values in Odoo Accounting
+   :alt: Example of a reconciliation model that suggests counterpart values in Odoo Accounting.
 
 .. _reconciliation_models_match:
 
 Match existing invoices/bills
 -----------------------------
 
-This type of Reconciliation Model automatically selects the right Customer Invoice or Vendor Bill
+This type of reconciliation model automatically selects the right customer invoice or vendor bill
 that matches the payment. All that is left to do is to validate the entry. This automation is based
 on a set of rules defined in the reconciliation model.
 
 .. image:: reconciliation_models/reconciliation_models_match.png
    :align: center
-   :alt: Example of a Reconciliation Model that matches existing invoices and bills automatically
-         in Odoo Accounting
+   :alt: Example of a reconciliation model that matches existing invoices and bills automatically
+         in Odoo Accounting.
 
 Configuration
 =============
 
-To manage or create new **Reconciliation Models**, go to :menuselection:`Accounting -->
-Reconciliation --> Reconciliation Models`. Alternatively, you can also open this menu from the
-Accounting Overview, by going to your Bank Journal card, clicking on the three little dots, and then
-on *Reconciliation Models*.
+To manage or create new **reconciliation models**, go to :menuselection:`Accounting dashboard -->
+Configuration --> Reconciliation Models`. Alternatively, you can also open this menu from
```

Comment:
True, it's because "Accounting" is also a menu within the accounting app and therefore if a user were to go to accounting and then configuration, he wouldn't find it, because configuration is a menu itself not part of the "accounting" menu.